src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,84 @@
+package duke.task;
+
+import java.time.format.DateTimeFormatter;
+
+/**
+ * This is a Task.
+ */
+public class Task {
+
+    public static final DateTimeFormatter DATE_TIME_FORMAT_IN = DateTimeFormatter.ofPattern("d/M/y kkmm");
+    public static final DateTimeFormatter DATE_TIME_FORMAT_OUT = DateTimeFormatter.ofPattern("MMM d y");
+    protected boolean isMarked = false;
+
+    protected String name;
+    /***
+     * Constructs a Task from the given name.
+     * @param name given name of the task.
+     */
+    Task(String name) {
+        this.name = name;
+    }
+    /***
+     * Void methods that sets the completeness of the task based on the input.
```

Comment:
Should the JavaDoc comment of the method start with a third-person present tense? e.g. `Sets the completeness of the task based on the input.`